lambdas/process-candidate-identity/src/main/java/uk/gov/di/ipv/core/processcandidateidentity/ProcessCandidateIdentityHandler.java

```diff
@@ -0,0 +1,297 @@
+            ConfigService configService,
+            ClientOAuthSessionDetailsService clientOAuthSessionDetailsService,
+            IpvSessionService ipvSessionService,
+            IdentityProcessingService identityProcessingService) {
+        this.configService = configService;
+        this.clientOAuthSessionDetailsService = clientOAuthSessionDetailsService;
+        this.ipvSessionService = ipvSessionService;
+        this.identityProcessingService = identityProcessingService;
+    }
+
+    @Override
+    @Tracing
+    @Logging
+    public Map<String, Object> handleRequest(ProcessRequest request, Context context) {
+        LogHelper.attachComponentId(configService);
+        configService.setFeatureSet(RequestHelper.getFeatureSet(request));
+
+        IpvSessionItem ipvSessionItem = null;
+        try {
+            var ipvSessionId = RequestHelper.getIpvSessionId(request);
+            var ipAddress = RequestHelper.getIpAddress(request);
+            var deviceInformation = request.getDeviceInformation();
+            var processIdentityType = RequestHelper.getProcessIdentityType(request);
+
+            ipvSessionItem = ipvSessionService.getIpvSession(ipvSessionId);
+            ClientOAuthSessionItem clientOAuthSessionItem =
+                    clientOAuthSessionDetailsService.getClientOAuthSession(
+                            ipvSessionItem.getClientOAuthSessionId());
+
+            String govukSigninJourneyId = clientOAuthSessionItem.getGovukSigninJourneyId();
+            LogHelper.attachGovukSigninJourneyIdToLogs(govukSigninJourneyId);
+
+            return switch (processIdentityType) {
+                case ALL -> processIdentityAll(
+                                ipvSessionItem,
+                                clientOAuthSessionItem,
+                                request,
+                                deviceInformation,
+                                ipAddress)
+                        .toObjectMap();
+                case COI -> processIdentityCoi(
+                                ipvSessionItem,
+                                clientOAuthSessionItem,
+                                request,
+                                deviceInformation,
+                                ipAddress)
+                        .toObjectMap();
+                case STORE_IDENTITY -> processIdentityStoreIdentity(
+                                ipvSessionItem,
+                                clientOAuthSessionItem,
+                                request,
+                                deviceInformation,
+                                ipAddress)
+                        .toObjectMap();
+                case TICF_ONLY -> processIdentityTicfOnly(
+                                ipvSessionItem,
+                                clientOAuthSessionItem,
+                                deviceInformation,
+                                ipAddress)
+                        .toObjectMap();
```

Comment:
I'm wasn't sure how to do this better. I wanted to abstract away as much of the processing logic as possible from the handler (which is why I added the `IdentityProcessingService` as an extra layer) so that the only bits here are worried about which processes to call and so keeping it more lightweight. However, passing the same args down multiple layers of functions feels like a code smell. I thought either to combine this inputs into an object (but it didn't feel right bc they were only related by the fact that they are shared inputs) or take away the `IdentityProcessingService` and move the functions there into the handler.